canvas_modules/common-canvas/__tests__/common-canvas/common-canvas-test.js

```diff
@@ -31,54 +33,102 @@ describe("CommonCanvas renders correctly", () => {
 		canvasController = new CanvasController();
 	});
 
+	it("should render one <CanvasBottomPanel/> component when showBottomPanel is true", () => {
+		const config = {};
+		const canvasRightPanelParams = {};
+		const wrapper = createCommonCanvas(config, canvasController, { showBottomPanel: true }, canvasRightPanelParams);
+		expect(wrapper.find(CanvasBottomPanel)).to.have.length(1);
+		expect(canvasController.isBottomPanelOpen() === true).to.be.true;
+	});
+
+	it("should render one <CanvasBottomPanel/> component when showBottomPanel is false", () => {
+		const config = {};
+		const canvasRightPanelParams = {};
+		const wrapper = createCommonCanvas(config, canvasController, { showBottomPanel: false }, canvasRightPanelParams);
+		expect(wrapper.find(CanvasBottomPanel)).to.have.length(1);
+		expect(canvasController.isBottomPanelOpen() === false).to.be.true;
+	});
+
+	it("should render one <CommonCanvasRightFlyout/> component when showRightFlyout is false", () => {
+		const config = {};
+		const canvasBottomPanelParams = {};
+		const wrapper = createCommonCanvas(config, canvasController, canvasBottomPanelParams, { showRightFlyout: false });
+		expect(wrapper.find(CommonCanvasRightFlyout)).to.have.length(1);
+		expect(canvasController.isRightFlyoutOpen() === true).to.be.false;
+	});
+
+	it("should render one <CommonCanvasRightFlyout/> component when showRightFlyout is true", () => {
+		const config = {};
+		const canvasBottomPanelParams = {};
+		const wrapper = createCommonCanvas(config, canvasController, canvasBottomPanelParams, { showRightFlyout: true });
+		expect(wrapper.find(CommonCanvasRightFlyout)).to.have.length(1);
+		expect(canvasController.isRightFlyoutOpen() === false).to.be.false;
+	});
+
 	it("should render one <CanvasContents/> component", () => {
 		const config = {};
-		const wrapper = createCommonCanvas(config, canvasController);
+		const canvasBottomPanelParams = {};
+		const canvasRightPanelParams = {};
+		const wrapper = createCommonCanvas(config, canvasController, canvasBottomPanelParams, canvasRightPanelParams);
```

Comment:
You shouldn't need to change the code here nor in any of the lines below, unless canvas parameter info is  is being passed in. See my comments below on createCommonCanvas.